### DISCUSSION TITLE
How to read large files?

### DISCUSSION THREAD

**Discussion by theomarkkuspaul** [gildas-lormeau/zip.js#425] [Q&A] [answered]
Hi there,

Looking to unzip large zip files using this package in the browser context. Passing a > 2GB Blob or Uint8Array has proven too much for my machine so looking into the docs for a better implementation I noticed this package supports split zip functionality. After some tinkering in the docs I couldn't quite figure out how to do it.

The [Zip Manager](https://gildas-lormeau.github.io/zip-manager/) seems to handle my large zip file well, but I couldn't quite reproduce your magic.

Would SplitDataReader do the job here? If so, do you have an example of how to implement this?

Any ideas here would be great :)

Thanks!

**Comment by gildas-lormeau** [comment, marked as answer]
Yous should be able to read a large zip file with the `BlobReader` class, do you confirm you're using this class?

**Reply by theomarkkuspaul** [reply]
I changed my implementation to use the `BlobReader` class instead, and it worked. Thanks for your response :)

**Reply by gildas-lormeau** [reply]
You're welcome, I'm glad to hear it :)
